models/netsuite/stg_netsuite__vendors.sql

```diff
@@ -36,6 +36,9 @@ final as (
         vendor_type_id,
         _fivetran_deleted
 
+        --The below macro adds the fields defined within your vendors_pass_through_columns variable into the staging model
+        {{ fivetran_utils.fill_pass_through_columns('vendors_pass_through_columns') }}
```

Comment:
Same comment as the other:

We will also want to add a similar macro the the respective `get_vendors_columns` macro. See how we did it for the accounts macro:
https://github.com/fivetran/dbt_netsuite_source/blob/799a3d9efc6d2ce258e35365d0b8f102f36ae24d/macros/get_accounts_columns.sql#L15

In particular you will want to add the following in the same place:
```yml
{{ fivetran_utils.add_pass_through_columns(columns, var('vendors_pass_through_columns')) }} 
```

Reply:
Done

Reply:
Done